### PR TITLE
v9.2 release

### PR DESCRIPTION
**Pull request by rvem**
## Description
The new Tezos release arrived, we're updating everything as usual.

Note, that native packages are waiting for opam-repository to be updated.

## Related issue(s)

Resolves None

#### Related changes (conditional)

- [x] I checked whether I should update the [README](../tree/master/README.md)

- [ ] I checked whether native packaging works, i.e. native binary packages
  can be successfully built.

#### Stylistic guide (mandatory)

- [x] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).

### CODE REVIEW

File: Formula/tezos-node-granadanet.rb [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+class TezosNodeFlorencenet < Formula
```
Comment (pasqu4le) [inline Formula/tezos-node-granadanet.rb:5]: Looks like this should be changed

**Review by pasqu4le — COMMENTED** [review]
Only one comment, also please use the `Problem`/`Solution` format in all the commits :slightly_smiling_face:

File: Formula/tezos-node-granadanet.rb (hunk 1 above)
Comment (rvem) [reply Formula/tezos-node-granadanet.rb:5]: right

Comment (rvem) [reply Formula/tezos-node-granadanet.rb]: Fixed, but I accidentally squashed changes with the latest commit:(

**Review by pasqu4le — APPROVED** [review]

**Pull request merged**